rfc/0084-app-service-level-telemetry.md

```diff
@@ -227,10 +227,6 @@ Once the server closes connection, the SDK should pick next available endpoint.
 Both initial and next endpoints must be randomized in case of more than one
 endpoint configured.
 
```

Comment:
I think we should keep something that mentions that we retry with backoff when the server closes the connection, in addition to when we don't receive a PONG in time (which is covered in the later paragraph).

My suggestion would be changing the wording of the above paragraph slightly:
> Once the server closes connection, **after backoff interval (see section below)**, the SDK should pick the next available endpoint, **and reconnect.**

Reply:
I agree. Will fix it